### DISCUSSION TITLE
《精灵宝可梦》第二世代游戏漏洞梳理

### DISCUSSION THREAD

**Discussion by giscus-xzonn** [Xzonn/xzonn.github.io#16] [General]
# 《精灵宝可梦》第二世代游戏漏洞梳理

《精灵宝可梦》第二世代游戏漏洞梳理 - 来自Xzonn的小站

https://xzonn.top/posts/Pokemon-Gen-2-Glitches.html

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****XXX 评论于 2023-02-12 20:56:08 (UTC+8)*****

請問水晶版是否有漏洞能產生色違的寶可夢？我試過金銀的任意代碼可以改變任意個體生成色違，但按版主的文章看來，水晶似乎只能產生任意寶可夢和道具，無法調整個體。
